test/orchard/java/parser_test.clj

```diff
@@ -0,0 +1,67 @@
+(ns orchard.java.parser-test
+  (:require [orchard.misc :as misc]
+            [clojure.test :refer :all]))
+
+(when (>= orchard.misc/java-api-version 9)
```

Comment:
Can you follow instead this pattern? https://github.com/clojure-emacs/orchard/commit/ffb71cd8935889a57b46a540f4c8b76df667272e

Reply:
Done, I added `test-newer-jdks` analogous to `src-newer-jdks`